samples_project/Assets/SampleViewer/Samples/FlightSim/readme.md

```diff
@@ -0,0 +1,29 @@
+# Third Person Character Controller
+
+Allows users to explore a city/map from the perspective of a Third Person Character.
+
+![Image of Flight Simulator](FlightSim.jpg)
+
+## How it works
+
+1. Check the box for **Mesh Colliers Enabled** on the `ArcGIS Map` component.
+2. Pick an airport to take off from and set the latitude and longitude of the airport as your world center
+3. Create a new blueprint and call it "bp_Runway".
+4. Add a cube and change the scale of it to be 50, 2500, 1. Or something similar that suits your needs.
```

Comment:
I think it should be better explained why the cube is needed and more info on where to place it. 
Ideally, the script should spawn the cube when the game begins instead of having the user add that

Reply:
That actually can be removed too. I copied and pasted the UE one into Unity but they are different. I just forgot to make the changes.

Reply:
Then maybe instead mention that they have to change the lat/lon of the runway in case they pick another airport?